### DISCUSSION TITLE
When using ctrl+c to end streaming, mediamtx exited and the console printed as follows. May I ask how to ensure that mediamtx does not end running?

### DISCUSSION THREAD

**Discussion by wycrystal** [bluenviron/mediamtx#2274] [Questions]
It's OK when I using gstreamer publish a stream to the server.
gst-launch-1.0 filesrc location=test.mp4 ! qtdemux name=d d.video_0 ! rtspclientsink protocols=tcp name=s location=rtsp://localhost:8554/mystream
When using ctrl+c to end streaming, mediamtx exited and the console printed as follows. May I ask how to ensure that mediamtx does not end running?
![image](https://github.com/bluenviron/mediamtx/assets/81356719/b72291bb-d6ab-4625-802a-7034e3420ef5)

**Comment by aler9** [comment]
Hello, this is a major bug that was fixed in https://github.com/bluenviron/gortsplib/pull/392, a new version will be released soon.

**Reply by wycrystal** [reply]
Thank you very much！

[a comment by an automated account is not shown]
